src/Normalizer/Relation/EmbedManyFieldNormalizer.php

```diff
@@ -51,6 +52,12 @@ public function normalizeField(
 
         $values = [];
         foreach ($childObjects as $i => $childObject) {
+            if (true === class_exists('Doctrine\Common\Persistence\Proxy')
```

Comment:
@TiMESPLiNTER please do not use true === if the return value of a method is a boolean